tests/mwe/test_mwe.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import pandas as pd
+from wordview.mwes.mwe import MWE
+
+
+@pytest.fixture
+def dummy_text_panda():
```

Comment:
`dummy_text_pandas`?

Reply:
You don't like the name of the fixture? I'm happy to change it. What would you consider a good name for the fixture?

Reply:
No it's good! I just wanted to suggest changing `panda` to `pandas`.